corehq/blobs/migratingdb.py

```diff
@@ -13,6 +13,7 @@ def __init__(self, new_db, old_db):
         self.new_db = new_db
         self.old_db = old_db
         assert new_db.metadb is not old_db.metadb
```

Comment:
@millerdev why is this important?

Reply:
because the very next statement (well not anymore) monkey patches `old_db.metadb.bulk_delete` with a no-op. If they were the same for some reason then `bulk_delete` of metadata would be completely disabled.

It's not expected that they would be the same, the assertion is just a sanity check.